frontend/packages/metal3-plugin/src/components/modals/StartNodeMaintenanceModal.tsx

```diff
@@ -41,7 +41,7 @@ const StartNodeMaintenanceModal = withHandlePromise<StartNodeMaintenanceModalPro
 
   const [cephClusters, loaded] = useK8sWatchResource<K8sResourceKind[]>(cephClusterResource);
   const cephCluster = cephClusters?.[0];
-  const cephClusterHealthy = !cephCluster || cephCluster?.status?.health === 'OK';
+  const cephClusterHealthy = !cephCluster || cephCluster?.status?.health === 'HEALTH_OK';
```

Comment:
```suggestion
  const cephClusterHealthy = cephCluster?.ceph?.status?.health === 'HEALTH_OK';
```

Reply:
thanks for pointing it out... just one thing we need this `!cephCluster` condition because we do not want to show the warning -alert if Ceph cluster is not installed.

Reply:
You are using optional chaining hence that would handle itself.
No worries if you want to keep it that way too.

Reply:
Do the respective change below:
```suggestion
!!cephCluster && !cephClusterHealthy &&
```